packages/template-retail-react-app/app/pages/account/order-detail.jsx

```diff
@@ -162,9 +229,9 @@ const AccountOrderDetail = () => {
                                     </Text>
                                     <Box>
                                         <Text fontSize="sm" textTransform="titlecase">
-                                            {shippingStatus.replace(/_/g, ' ')}
+                                            {shippingStatus?.replace(/_/g, ' ')}
                                         </Text>
-                                        <Text fontSize="sm">{shippingMethod.name}</Text>
+                                        <Text fontSize="sm">{shippingMethod?.name}</Text>
```

Comment:
Why are these all changed to optional? Would there be a jump when the data loads?

Reply:
Yep, making them optional is not necessary. It was code from the initial attempt to just get things working using hooks.